src/averageAges.js

```diff
@@ -1,61 +1,64 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+// function to find only men including century specifying
+
+function filterMen(people, sex, century) {
+  return people.filter(person => {
+    return person.sex === sex && century
+      ? century === Math.ceil(person.died / 100)
+      : person.sex === sex;
```

Comment:
You can create two variables:
`isMan` - check person sex;
`isSameCentury` - check century with ternary;

And return combined `isMan && isCentury`.